tests/cli-configuration.spec.js

```diff
@@ -265,4 +266,14 @@ describe('CLI Configuration', function() {
 
   });
 
+  describe('.load()', function() {
+
+    it(`should load defualt internal ${packageJSON.name}.conf.js if file is not found`, function() {
```

Comment:
`default` typo.

Also, does this change mean that if the config-file that the user specifies doesn't exist, versionist will silently load its internal config file, rather than error that the user specified a non-existent filename? IMHO that'd be undesired.